lib/tasks/braintree_tokens.rake

```diff
@@ -0,0 +1,8 @@
+namespace :champaign do
+  desc "Create braintree payment method tokens out of existing token columns"
+  task braintree_tokens: :environment do
+    Payment::BraintreeCustomer.all.each do |customer|
+      Payment::BraintreePaymentMethod.create!({customer_id: customer.customer_id, token: customer.card_vault_token})
```

Comment:
ahh got it. OK I think we just need to make sure that `schema.rb` reflects what will actually happen if someone who hasn't used your branch runs the migrations